src/masternodes/changiintermediates.h

```diff
@@ -0,0 +1,23 @@
+#ifndef DEFI_MASTERNODES_CHANGIINTERMEDIATES_H
+#define DEFI_MASTERNODES_CHANGIINTERMEDIATES_H
+
+#include <consensus/params.h>
```

Comment:
Include should be in the cpp file with forward declarations here instead. Includes in headers should be as minimal as possible to avoid include loops.

Reply:
Oh right, my bad.. missed this